assets/js/modules/adsense/index.js

```diff
@@ -113,4 +108,10 @@ domReady( () => {
 			AREA_DASHBOARD_EARNINGS,
 		],
 	);
+	Modules.registerModule(
+		'adsense',
+		{
+			setupComponent: SetupMain,
+		},
+	);
```

Comment:
Super-nitpick, but maybe we should put the `registerModule` call above the widget registrations, just because module registration is the more general step. Of course this doesn't technically change anything, but without module registration widget registration is mostly useless (most widgets only are active if their respective module is registered and active).

This also applies to the other similar files.